slides/rights_and_licenses.md

```diff
@@ -0,0 +1,95 @@
+---
+title:  Rights and Licenses
+author: Anni Järvenpää
+date:   2023-06-20
+lang:   en
+---
+
+
+# In This Session
+
+- What kinds of licenses can resources offered via the Language Bank have?
+- How to access the resources?
+- What you can do with them?
+- What about very restricted data?
```

Comment:
Or "sensitive data"?

Reply:
I chose not to use the word "sensitive" here, as depending on the context sensitive data can mean something along the lines of "any data that needs to be protected against unauthorized access" which triggers a lot before we get to SD desktop level needs. Though "restricted" is maybe too much too, as I'm guessing that our near-future data sets won't be ST IV either.

Maybe "very sensitive" would be better?

Reply:
Hang on, is this intended to be a lead into RES corpora, or into SD desktop?

Reply:
SD desktop

Reply:
Okay, then definitely very sensitive is better than very restricted.